gatorgrouper_django/gatorgrouper/migrations/0001_initial.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 2.1.7 on 2019-02-15 21:15
```

Comment:
Since the Django script can generate this file automatically, we don't need to commit it IMO

Reply:
Added gatorgrouper_django/gatorgrouper/migrations/ to .gitignore